submissions/akulahere/html-css-popup/css/main.css

```diff
@@ -0,0 +1,138 @@
+body {
+    font-family: "Times New Roman";
+    margin: 0px;
+}
+
+a {
+    text-decoration: none;
+    color:black;
+}
+
+img {
+    height: 30px;
+    width: 30px;
+}
+
+ul {
+    list-style-type: none;
+    padding-inline-start: 0px;
+}
```

Comment:
Be better if for css you would be use classes.

Reply:
> Be better if for css you would be use classes.

That's common styles, and in this case, it's needlessly.

Reply:
I know and if I correctly understand, he can download this file without comments